go/host/node/node.go

```diff
@@ -39,6 +39,9 @@ const (
 	apiNamespaceObscuroScan = "obscuroscan"
 	apiNamespaceNetwork     = "net"
 	apiNamespaceTest        = "test"
+
+	l1TxRetriesRollup = 3
```

Comment:
some comments would be helpful here describing what these constants are and how much time that means

Reply:
I'll add a comment. It's hard to say accurately with timing, as on top of the pauses, there's the duration of the request itself.

Reply:
of course. just ballpark.

Reply:
👍